data/json/items/gun/223.json

```
@@ -132,6 +132,69 @@
     ],
     "melee_damage": { "bash": 9 }
   },
+  {
+    "id": "plr16",
+    "copy-from": "rifle_semi",
+    "looks_like": "skorpion_61",
+    "type": "GUN",
+    "name": { "str": "Kel-Tec PLR16" },
+    "description": "Semi-automatic pistol, chambered in 5.56 NATO, using gas piston to reduce lenght and frame constructed from reinforced polymer to make the it both durable and light-weight. This one comes with plastic handguard.",
```

```suggestion
    "description": "This is a semi-automatic pistol chambered in 5.56 NATO, using a gas piston to reduce length and a frame constructed from reinforced polymer to make it both durable and light-weight.  This one comes with a plastic handguard.",
```

---

```suggestion
    "description": "A semi-automatic pistol, chambered in 5.56 NATO, using a gas piston to reduce length and a frame constructed from reinforced polymer to make the it both durable and lightweight. This one comes with a plastic handguard.",
```
We don't generally do start with "this" in item descriptions, as you're obviously looking at whatever the thing is.
(See "Item Description Cleanup Project" PRs by FoolsGold45)